build.gradle

```diff
@@ -51,7 +51,7 @@ ext {
 
 dependencies {
     implementation 'com.google.code.gson:gson:2.8.9'
-    implementation 'org.mifos:ph-ee-connector-common:1.4.1-SNAPSHOT'
+    implementation 'org.mifos:ph-ee-connector-common:1.7.2-rc.2'
```

Comment:
Use 1.8.1 snapshot version here. 
If there are unmerged changes in connector common please let me know

Reply:
Updated the version